translations/messages+intl-icu.de.yml

```diff
@@ -1277,6 +1277,7 @@ error.statement.move.unownedprocedure: "Stellungnahme kann nicht verschoben werd
 error.statement.new: "Beim Anlegen der Stellungnahme ist ein Fehler aufgetreten."
 error.statement.not.found: "Die Stellungnahme konnte nicht gefunden werden"
 error.statement.not.saved: "Die Stellungnahme konnte nicht gespeichert werden, bitte überprüfen Sie noch einmal alle Eingabefelder."
+error.statement.not.saved.throttle: "Die Stellungnahme konnte nicht gespeichert werden, da das Formular zu häufig abgeschickt wurde. Bitte warten Sie einen Moment."
```

Comment:
Did you intentionally keep it vague to not expose the actual rate? In case users mistakenly sended an unfinished statement this message misses the critical information "when exactly will i be able to resume". In fact i feel that 15 minutes are more then "one moment".

Reply:
Yes I did it intentionally vague. When you think it is necessary from a ux point of view I can include the minutes in the message

Reply:
I think if somebody wanted to tweak a scripted "statement spamming software" to run every x times, they would be able to find out the rate limit by trial and error anyway. Manually spanning the system is not possible either, as the rate makes that inefficient. Thats why i'd argue we can also expose the actual rate in the message. On the other hand, the case i made up (unintentionally sending an unfinished statement) is indeed a little bit edgy. I'd leave it up to you to decide it :)

Reply:
It is possible to add the specific time but this would require to set specific headers to the API Response (which is possible) and some bigger changes in the StatementModal to react on those headers to pass them into the translation. Do you think this is worth it?

Reply:
> Do you think this is worth it?

After all that has been said: no.